core/cmake/BareosCheckIncludes.cmake

```diff
@@ -60,7 +60,14 @@ check_include_files(sys/ea.h HAVE_SYS_EA_H)
 check_include_files(sys/proplist.h HAVE_SYS_PROPLIST_H)
 check_include_files(sys/xattr.h HAVE_SYS_XATTR_H)
 
-check_include_files("sys/stat.h;cephfs/ceph_statx.h" HAVE_CEPHFS_CEPH_STATX_H)
+
+include(CheckSymbolExists)
+include(CMakePushCheckState)
+cmake_push_check_state()
+set(CMAKE_REQUIRED_LIBRARIES cephfs)
+check_symbol_exists(ceph_statx "sys/stat.h;cephfs/libcephfs.h" HAVE_CEPHFS_CEPH_STATX_H)
```

Comment:
The name `HAVE_CEPHFS_STATX_H` is now misleading. Maybe we could rename it `HAVE_CEPHFS_STATX`?